t/001-wildcard-router.t

```diff
@@ -44,13 +52,24 @@ credentials missing!
 
 --- main_config
 env RESOLVER=127.0.0.1:1953;
+env API_HOST=http://localhost:8081;
+env MASTER_ACCESS_TOKEN=alaska-token;
 --- http_config
   lua_package_path "$TEST_NGINX_LUA_PATH";
   include $TEST_NGINX_UPSTREAM_CONFIG;
+  server {
+    listen 8081;
```

Comment:
It is possible to reuse the port Test::Nginx starts on by `$TEST_NGINX_SERVER_PORT`